src/server/routes/revision.js

```diff
@@ -169,12 +169,14 @@ function diffRevisionsWithParents(revisions) {
 				.then(
 					(parent) => Promise.props({
 						changes: revision.diff(parent),
-						entity: revision.related('entity')
+						entity: revision.related('entity'),
+						entityAlias: revision.related('data').fetch({require: false, withRelated: ['aliasSet.defaultAlias']})
```

Comment:
Instead of fetching here, you can add `'data.aliasSet.defaultAlias'` to withRelated in `_createRevision` and just use `revision.related('data')` as we do for `entity` just above.

Should be replicated for the identical promise below.